README.rst

```diff
@@ -35,6 +35,8 @@ For a comprehensive list all of the interactive Jupyter Notebooks in our repo (i
 
 For the latest news regarding Cirq, sign up to the `Cirq-announce email list <https://groups.google.com/forum/#!forum/cirq-announce>`__!
 
+**Note:** `cirq-ft` is no longer maintained. Please use `Qualtran` as an alternative for similar functionalities.
```

Comment:
Want to add a link?  Otherwise, LGTM.

Reply:
@dstrain115 This is not the right place for this, plus the Cirq README is about to change. I think we should close the PR, since the OP has not responded for 2 weeks.